cluster/common.sh

```diff
@@ -160,6 +160,28 @@ function clear-kubeconfig() {
   echo "Cleared config for ${CONTEXT} from ${KUBECONFIG}"
 }
 
+# Creates a kubeconfig file for the kubelet.
+# Args: address (e.g. "http://localhost:8080"), destination dir
+function create-kubelet-kubeconfig() {
```

Comment:
What's the point of having this in common.sh if it's copy pasted everywhere? Common.sh ususally doesn't contain functions that run remotely. it contains functions that run on the computer where you run kube-up.sh